src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberContainerSymbol.cs

```diff
@@ -4679,6 +4689,21 @@ internal static bool IsAllowedExtensionMember(Symbol member)
             return false;
         }
 
+        internal static bool IsAllowedExtensionMember(Symbol member, LanguageVersion languageVersion)
+        {
+            if (IsAllowedExtensionMemberInCSharp14(member))
```

Comment:
>IsAllowedExtensionMemberInCSharp14 [](http://example.com/codeflow?start=16&length=34)

It looks like this is the only use site for this method. Consider merging two methods, or, in other words, modifying the original method instead of splitting it.